synapse/lib/config.py

```diff
@@ -392,28 +392,16 @@ def reqConfValid(self):
         else:
             return
 
-    def reqConfValu(self, key):
+    def req(self, name):
```

Comment:
This breaks downstream users

Reply:
I'll plumb a depr passthrough so we can start a clock on making the change for real :+1: